src/libs/ajax/leonardo/models/disk-models.ts

```diff
@@ -61,12 +63,24 @@ export interface ListDiskItem {
 export type PersistentDisk = ListDiskItem | GetDiskItem;
 export type AppDataDisk = PersistentDisk;
 
+// TODO: Rename
 export interface PdType {
+  // TODO: GoogleDiskType is not a label, it is a value
   label: GoogleDiskType;
   displayName: string;
   regionToPricesName: string;
+  // cloudProvider: 'GCP';
```

Comment:
i don't think this needs `cloudProvider`, seems redundant with the `cloudContext` in the disk itself